test/e2e/config/e2e_conf.yaml

```diff
@@ -144,7 +144,22 @@ providers:
       targetName: "cluster-template-ubuntu.yaml"
     - sourcePath: "../_out/cluster-template-upgrade-workload.yaml"
       targetName: "cluster-template-upgrade-workload.yaml"
-  - name: v1.7.99
+  - name: "{go://github.com/metal3-io/cluster-api-provider-metal3@v1.8}"
```

Comment:
Can you add this logically on top, now it is sorted as 1.7, 1.6, 1.5, 1.8

Or rather, since the 1.9 is the lowest, sort them as 1.5 1.6 1.7 1.8 main(1.9)

Reply:
fixed